### ISSUE TITLE
Generate the style.yml from tileJSON and metadata.json scheme

### ISSUE THREAD

**Issue by miya0001** [unvt/charites#20]
* https://github.com/mapbox/tilejson-spec
* https://github.com/mapbox/mbutil

**Comment by JinIgarashi** [comment]
In addition, PR #27 just implemented to create default layers on a single `style.yml` inside. However, in the future, we may add another option to generate configuration files separately for each layers.

Decomposing layer configuration files can be like the following command.
```
charites init style.yml --tilejson-urls <--tilejson-urls> --decomposite-layers
```

In root style.yml, it indicate the location of each layer configuration files.
```
layers:
  - !!inc/file layers/background.yml
  - !!inc/file layers/water.yml
```

**Comment by miya0001** [comment]
Thanks 😊 

If you load multiple TileJSONs, then  many layers will be loaded.
The user will want to adjust the order of those layers, and it is better if those layers have separated files.

And I don't want to add options if possible to keep easy to use.

**Comment by JinIgarashi** [comment]
@miya0001 Sure, It makes sense to generate layers file separately for convenience of users. Let me try working on #27.

As default, it can create separately as following command.

```
charites init style.yml --tilejson-urls <--tilejson-urls>
```

if `--composite-layers` option is added, it will generate layers in a single root file.

Multiple tilejson importing was already implemented, you can use comma to combine multiple URL.

**Comment by JinIgarashi** [comment]
@miya0001 I have implemented in #27. Please check it.

**Comment by JinIgarashi** [comment]
close this issue now
